packages/app/templates/ui-extensions/projects/checkout_ui_beta/shopify.ui.extension.toml.liquid

```diff
@@ -0,0 +1,25 @@
+name = "{{ name }}"
+
+[capabilities]
+network_access = true
+block_progress = true
+
+[[extension_points]]
+target = "Checkout::Dynamic::Render"
+module = "./src/checkoutDynamicRender.{{ srcFileExtension }}"
```

Comment:
On the "final nitpicking" front: assuming we like the "file name is extension point without `::`s", I prefer pascal case. It matches the extension point, and for React consumers I think the pascal case name is pretty conventional.